docs/reference/commandline/build.md

```diff
@@ -30,6 +30,7 @@ weight=1
       --cpuset-cpus=""         CPUs in which to allow execution, e.g. `0-3`, `0,1`
       --cgroup-parent=""       Optional parent cgroup for the container
       --ulimit=[]              Ulimit options
+      --help=false             Print usage
```

Comment:
you know this is a major effort to do, will instead address this in another PR wdyt? at least now we have the help flag :)